source/xlld/wrap/traits.d

```diff
@@ -374,7 +374,7 @@ void generateDllDef(string module_ = __MODULE__,
  */
 enum Async;
 
-version(unittest) {
+version(testingExcelD) {
```

Comment:
Why would dependent libraries _want_ to define their own `getWorksheetFunctions`?